static-site/content/docs/07-visualisation/02-map-interpretation.md

```diff
@@ -0,0 +1,98 @@
+In Fig 2C we have a discrete color scale representing authorship, and we convey this information by a pie chart with areas of each slice representing the proportion of samples in this location from each author.
+In Fig 2D we have a continuous color scale (sampling date) and convey this by a blending of colors -- the samples which make up this deme actually span Jun 2016 and Oct 2016.
+You can see the legend representing in currently selected metadata in either the top-left of the phylogenetic tree or the top-right of the map (see "3" in Figure 1).
+
+
+The **location** of circles represents the geographic resolution currently selected.
+Looking at Figure 1, we have a geo-resolution of Country (labelled "5") and so each circle represents an individual country.
+In the [zika dataset](https://nextstrain.org/zika) you can change this to "Region" and you'll see the map update accordingly.
+Each dataset is responsible for associating different locations with a GPS co-ordinate which we use as the center of the circle -- bear this in mind when you are interpreting the map and don't over-assign precision.
+For example, the circle representing the USA (labelled "1" in Figure 1) represents the entire country, not a specific location in Kansas!
+
+
+## Transmissions (Lines)
+
+These lines typically represent our best estimate of the movement of the pathogen over the selected time slice.
+For instance, the line indicated by "2" in Figure 1 indicates inferred movement of Zika virus from French Polynesia to Brazil.
+Each individual line represents a parent-child branch in the phylogenetic tree where the value of the current geographic-resolution differs.
+This means that if values for the geographic-resolution haven't been inferred for internal nodes of the tree then no transmission lines can be drawn.
+There is a toggle to switch these off in the sidebar (Figure 1, label "5"). 
+
+![](./img/labelled-movement.png)
+_Figure 3.
+Left: Cartoon of how the tree determines the construction of transmission lines.
+Right: Example of transmission lines showing [between country movement of Ebola](https://nextstrain.org/ebola)._
+
+**Line thickness.**
+Each rendered arc is actually a collection of individual lines of identical thickness (as above, each line identified by a branch in the tree indicating a change in the value of the current geographic resolution).
+Lines which look "thicker" therefore represent more observed cases of movement between the two locations (circles).
+For instance, in figure 3 the arc labelled by "1" is thicker than that labelled "3", indicating that in the [Ebola dataset](https://nextstrain.org/ebola) there are more observed cases of movement between Guinea and Sierra Leone than there are between Liberia and Guinea.
+
+
+**Directionality** is conveyed by the [orientation of the arc](https://en.wikipedia.org/wiki/Curve_orientation).
```

Comment:
I don't grasp this very well even after reading the wikipedia article. Maybe we can chat on the phone about how you commonly explain this to people orally (e.g. while giving a presentation of nextstrain) and that will help inform any suggestions I would have for this section.

Reply:
👍 rewritten this section to use language such as "the inside of the curve is on the left-hand-side"